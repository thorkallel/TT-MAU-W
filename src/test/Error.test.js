import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

import CriptoState from "../context/cripto/criptoState";
import CotizacionState from "../context/cotizacion/cotizacionState";

import Error from "../components/Error";

describe("Pruebas en <Error />", () => {
  test("debe mostrar <Error /> correctamente", () => {

    const wrapper = shallow(
      <CriptoState>
        <CotizacionState>
          <Error />
        </CotizacionState>
      </CriptoState>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
