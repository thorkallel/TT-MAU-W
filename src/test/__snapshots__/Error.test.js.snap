// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Error /> debe mostrar <Error /> correctamente 1`] = `
<ContextProvider
  value={
    Object {
      "criptomoneda": "",
      "datacripto": Array [],
      "fGetDatacripto": [Function],
      "fSaveCripto": [Function],
      "loadstate": true,
    }
  }
>
  <CotizacionState>
    <Error />
  </CotizacionState>
</ContextProvider>
`;
